tests/server_event.rs

```diff
@@ -244,6 +244,58 @@ fn event_queue() {
     assert_eq!(client_app.world().resource::<Events<DummyEvent>>().len(), 1);
 }
 
+#[test]
+fn independent_event_queue() {
```

Comment:
The event won't be queued
```suggestion
fn independent_event() {
```